lib/features/base/mixin/contact_support_mixin.dart

```diff
@@ -0,0 +1,38 @@
+
+import 'package:core/utils/app_logger.dart';
+import 'package:jmap_dart_client/jmap/mail/email/email_address.dart';
+import 'package:model/extensions/contact_support_capability_extension.dart';
+import 'package:model/support/contact_support_capability.dart';
+import 'package:tmail_ui_user/features/email/presentation/model/composer_arguments.dart';
+import 'package:tmail_ui_user/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart';
+import 'package:tmail_ui_user/main/utils/app_utils.dart';
+
+typedef OnTapContactSupportAction = Function(ContactSupportCapability contactSupport);
+
+mixin ContactSupportMixin {
+
+  void onGetHelpOrReportBug(
+      ContactSupportCapability contactSupport,
+      MailboxDashBoardController mailboxDashBoardController,
+  ) {
+    log('ContactSupportMixin::onGetHelpOrReportBug:contactSupport = $contactSupport');
+    if (contactSupport.isMailAddressSupported) {
+      _handleMailAddress(contactSupport.supportMailAddress!, mailboxDashBoardController);
```

Comment:
Should we return after this? For safe handling?

Reply:
No, we have done `if/else` condition checking here. So if `if` is already entered it will not enter `else if` again.

Reply:
<img width="646" alt="Screenshot 2024-12-12 at 10 11 34" src="https://github.com/user-attachments/assets/24cda710-ebea-49d8-bc06-af983a8564fe" />